development/README.md

```diff
@@ -0,0 +1,25 @@
+# Development
+This folder is only for developing and testing the SDK and should not need to be touched by anyone outside of the LUCI organization.
+
+LUCI engineers use this Docker image to develope and test new ROS packages.
+
+This Docker image is also used by the individual package repos for their CI/CD actions.
+
+# Building
+
+# Using
+
+### SSH key transfer
+In order to clone the LUCI repos you will need an ssh key on the Docker image you have running. The easiest way to do this it to copy over and existing key from your dev machine. 
```

Comment:
Might be worth mentioning that there are other ways to do this such as https://www.jamesridgway.co.uk/sharing-an-ssh-agent-between-a-host-machine-and-a-docker-container/ and
https://gist.github.com/d11wtq/8699521 which don't require copying the keys but just use mounts and the agent.

Reply:
Those look better Ill flesh out the instructions on the second PR going in a few days thanks!